_sass/components/_card.scss

```diff
@@ -109,17 +113,36 @@ img.card-image {
 
 .card-content {
   padding: $base-padding;
-}
 
-.card-title {
-  margin-bottom: $base-padding-lite;
-  padding-right: $base-padding;
+  p {
+    @include heading(para-md);
+    color: darken($neutral-gray, 20%);
+    line-height: 1.6;
+  }
 
-  a {
+  .card-content-subhead {
     color: $dark-gray;
+    font-weight: $weight-bold;
+    margin-bottom: $base-padding-lite;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 129 to appear before line 117. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets